content/qa/automation-testing-basic-guide.md

```diff
@@ -0,0 +1,38 @@
+### Automation Testing Basic Guide
+
+Before jump into any test tool, let's go through some basic concept automation testing. If you are already aware about few things listed below then please free to move to the next chapter.
+
+1. **What is Automation Testing?**
+   The test scenario which is executed by some tool is called Automation testing.
+2. **Why Automation Testing is needed? Or Goal of using Automation Testing.**  
+   There are many points which we can consider that why automation testing needed or it is important.
+
+   * The basic reason is that It does not require manual intervenation.
+   * It is easy to run the automation script for a multilingual sites then testing it manually
+   * Increases the speed of testing
+   * Manual Testing can become boring and hence error-prone.
+
+3. **Is Automation Testing is an alternate to Manual Testing?**  
+   No. It is not an alternate to manual testing. Since every test scenario can not be tested automatically. There are certain creiteria to automate the test cases. If something does not fulfill the scope of automation then it can not be automated. And hence we can not completely relay on Automation Testing.
+
+4. **Which test cases should be automated?**
+
+   **We can consider the below scenarios where the test cases can be automated. **
```

Comment:
Remove space before `**` to fix markup issue.

`**We can consider the below scenarios where the test cases can be automated.**`

Reply:
Done

Reply:
Its not. still have space before `**`